west.yml

```diff
@@ -261,7 +261,7 @@ manifest:
     - name: hostap
       repo-path: hostap
       path: modules/lib/hostap
-      revision: 83574f533fb5c6808af0d9872741d72d48be2f98
+      revision: pull/10/head
```

Comment:
This can be changed to 20f4cacf5a69eed9bc684f11c62b8f75349f223a as that is now the latest version of hostap repo.